dev/PowerNotifications/PowerNotifications.h

```diff
@@ -107,21 +107,21 @@ namespace winrt::Microsoft::Windows::System::Power
         {
             std::mutex m_mutex;
             int m_batteryChargePercent{ 100 };
-            int m_oldBatteryChargePercent;
-            DWORD m_cachedDisplayStatus;
-            DWORD m_cachedUserPresenceStatus;
-            DWORD m_cachedSystemAwayModeStatus;
-            DWORD m_cachedPowerSourceKind;
-            EFFECTIVE_POWER_MODE m_cachedPowerMode;
-            ULONGLONG m_cachedDischargeTime;
+            int m_oldBatteryChargePercent{ 0 };
+            DWORD m_cachedDisplayStatus{ 0 };
+            DWORD m_cachedUserPresenceStatus{ 0 };
+            DWORD m_cachedSystemAwayModeStatus{ 0 };
+            DWORD m_cachedPowerSourceKind{ 0 };
+            EFFECTIVE_POWER_MODE m_cachedPowerMode = EffectivePowerModeBatterySaver;
```

Comment:
Nit: Is there a reason why this cannot use "{ EffectivePowerModeBatterySaver };" for consistency?